Framework/Algorithms/src/ApplyTransmissionCorrection.cpp

```diff
@@ -91,7 +90,7 @@ void ApplyTransmissionCorrection::exec() {
   Progress progress(this, 0.0, 1.0, numHists);
 
   // Create a Workspace2D to match the intput workspace
-  MatrixWorkspace_sptr corrWS = create<MatrixWorkspace>(*inputWS);
+  MatrixWorkspace_sptr corrWS = WorkspaceFactory::Instance().create(inputWS);
```

Comment:
Try `create<HistoWorkspace>`, otherwise the new factory keeps the event workspace, which may have been the source of the issue (if the input was events).